src/stdune/hashtbl_intf.ml

```diff
@@ -1,7 +1,9 @@
 module type S = sig
   include MoreLabels.Hashtbl.S
 
-  val add : 'a t -> key -> 'a -> unit
+  val set : 'a t -> key -> 'a -> unit
+
+  val add_exn : 'a t -> key -> 'a -> unit
```

Comment:
Don't you want a version of `add` that returns a `Result`, by the way? I think that will be useful e.g. to restore custom error messages.

Reply:
Indeed. I was going to add it but there's nowhere to use it yet, and I wanted to see what you thought of the interface i proposed.